docs/bin/build-and-mirror.sh

```diff
@@ -20,6 +20,7 @@ function build_and_mirror {
 	set +x
 	if [ "$release" == "root" ] ; then
 		set -x
+		for file in $(ls $GEOMESA/docs/target/html/); do rm -r $GEOMESA_GITHUB_IO/documentation/$file; done
```

Comment:
Just to be clear, this doesn't do anything with the versioned documentation?

Reply:
right - they're all in the same folder, so it's a bit tricky to not modify them

Reply:
I played around with rsync --include and --exclude but couldn't really get anything to work right